tests/ctst/steps/utils/utils.ts

```diff
@@ -199,6 +199,83 @@ async function createBucketWithConfiguration(
     }
 }
 
+async function putMpuObject(world: Zenko, parts: number = 2, objectName: string, content?: string) {
```

Comment:
Isn't there a minimum of size for parts of an MPU (IIRC its 5MB) ?. If so, should we log an error here if the test is misswritten or should we let S3 respond with an error ?

Reply:
We should let the test fail, actually the SDK may be already doing the check for us.

Reply:
not sure we have this limitations...
test seems to pass, so we should be fine... unless we silently ignore the s3 error, as there is no assertion after `completeMpuObject` :-/

Reply:
We do have this limitation in cloudserver when doing a PutPart, but it does not consider the last part. This is only checked during CompleteMPU:

```

    // Note: Part sizes cannot be less than 5MB in size except for the last.
    // However, we do not check this value here because we cannot know which
    // part will be the last until a complete MPU request is made. Thus, we let
    // the completeMultipartUpload API check that all parts except the last are
    // at least 5MB.
```